std_msgs/String color
---
visualization_msgs/MarkerArray block_poses

